main.rb

```diff
@@ -0,0 +1,40 @@
+require_relative './app'
+
+def request
+  puts 'Please choose an option below :'
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
+  puts '6 - List all rentals for a given person id'
+  puts '7 - Exit'
+end
+
+puts 'Welcome to School Library App!'
+
+def actions
+  case input_actions
+  when '1'
+    list_all_books
+  when '2'
+    list_all_people
+  when '3'
+    create_person
+  when '4'
+    create_book
+  when '5'
+    create_rental
+  when '6'
+    list_all_rental
+  else
+    puts 'Thank you for using this app'
+  end
+end
+
```

Comment:
- [ ] I am getting errors while running the app. I would recommend what the previous reviewer suggested. Put all these inside `main` method so that it will run after we call `main` method.


The error I'm getting

![Screenshot from 2022-07-22 21-19-43](https://user-images.githubusercontent.com/26767383/180477531-4cf9c81f-955c-4607-9be0-c6a888196dd4.png)

Hint to fix:

```ruby
def main
  app = App.new()

  response = nil

  puts "Welcome to School Library App!\n\n"

  while response != "7"
    puts "Please choose an option by enterin a number:"
    puts "1 - List all books"
    puts "2 - List all people"
    ...
    ...
    puts "7 - Exit"
    
    # *Lets users choose an option.
    response = gets.chomp

    case response
    when "1"
      app.list_books()
    when "2"
      app.list_people()
    ....
    ....
    when "7"
      puts "Thank you for using this app!"
    end

    puts "\n"
  end
end

main()

```

Reply:
@Deepakdanger
Thanks for your suggestions but when I implemented the changes as the first reviewer requested, I get rubocop errors. And am getting the errors again as I implemented it the way you suggested . This is the error I get "(Metrics/CyclomaticComplexity: Cyclomatic complexity for main is too high. [9/7]
def main ...
^^^^^^^^ and  Metrics/MethodLength: Method has too many lines. [31/20]
def main ...)"